pulsar-client-cpp/lib/ClientConnection.h

```diff
@@ -267,6 +267,9 @@ class ClientConnection : public boost::enable_shared_from_this<ClientConnection>
     bool havePendingPingRequest_;
     DeadlineTimerPtr keepAliveTimer_;
 
+    unsigned long maxPendingLookupRequest_;
```

Comment:
Use type with the size info, e.g.: either uint32_t or uint64_t. In this case 32 bits should be enough